onnxruntime/contrib_ops/cpu/cdist.cc

```diff
@@ -10,7 +15,94 @@ namespace contrib {
                                 KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<data_type>()), \
                                 CDist<data_type>);
 DEFINE_KERNEL(float);
-DEFINE_KERNEL(double);
+//DEFINE_KERNEL(double);
+
+template <typename T>
+static void cdist_euclidian_onshot(const Tensor& a, const Tensor& b, Tensor& c, concurrency::ThreadPool* threadpool) {
```

Comment:
This was meant to be called 'oneshot', but after cleanup it's the only remaining thing so will rename tomorrow when adding support for double.